.github/workflows/lock-threads.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Lock Closed Threads'
+
+on:
+  schedule:
+    - cron: '0 7 * * *'  # 7am UTC, daily
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+concurrency:
+  group: lock
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          issue-lock-inactive-days: '30'
+          pr-lock-inactive-days: '15'
```

Comment:
Do you want the workflow to apply labels? If you want, you should add the following options:

```yaml
    issue-lock-labels: "S: auto-locked"
    pr-lock-labels: "S: auto-locked"
```

Reply:
Nah, I think showing up as locked by Github Actions is sufficient.

Reply:
Ok, no problem.